conftest.py

```diff
@@ -1,14 +1,134 @@
 from django.contrib.auth.models import User
-from account.models.profile import Profile
+from account.models.profile import Profile, UserType
 from account.models.professional import Professions, Professional
 from account.models.client import Client
 from chatmessage.models import Chatmessage, SenderType
 from datetime import datetime
 import pytest
 
-BIRTHDAY = datetime(2000, 1, 1)
+
+USERNAME = "testusername"
+PASSWORD = "testpassword"
+FIRST_NAME = 'Bob'
+LAST_NAME = 'Builder'
+EMAIL = "test@test.com"
 LAST_LOGIN = datetime.now()
 
+PROFILE_TYPE = 'C'
+PHONE_NUMBER = '123456789'
+COUNTRY = 'USA'
+CITY = 'New York'
+ADDRESS = '123 Main St'
+
+PROFESSION = Professions.Locksmith
+DESCRIPTION = 'Test Description'
+
+BIRTHDAY = datetime(2000, 1, 1)
+
+
+@pytest.fixture
+def make_user():
+    def make(
+        username: str = USERNAME,
+        password: str = PASSWORD,
+        first_name: str = FIRST_NAME,
+        last_name: str = LAST_NAME,
+        email: str = EMAIL,
+        last_login: datetime = LAST_LOGIN
+    ):
+        user = User(
+            username=username, password=password, first_name=first_name,
+            last_name=last_name, email=email, last_login=last_login
+        )
+        user.save()
+        return user
+
```

Comment:
please delete line 43 
and change line 39 to be User.objects.create_user

Reply:
Done @idosinger98